about page not updating..?

bethanyhaubert opened the pull request.

bethanyhaubert committed:
commit

bethanyhaubert committed:
strikethrough fix

bethanyhaubert committed:
fixing format

bethanyhaubert committed:
formatting ss

bethanyhaubert committed:
formatting errors

bethanyhaubert committed:
formatting erros

bethanyhaubert committed:
format errors

bethanyhaubert committed:
again

bethanyhaubert committed:
again

bethanyhaubert committed:
title formatting

bethanyhaubert committed:
again

bethanyhaubert commented:
The info in the 'about' page is in 'index.html'. I replaced the existing info with mine and it's not showing. tried it in a few places on that page to test.

sumeetjain commented:
I'm not sure I follow you. Are you saying that you want the homepage of the entire site to be the About Page?

Or that you put the content for the About page into the index.html page? If so, I cannot see it either. Might be best to come up.

bethanyhaubert committed:
about

bethanyhaubert committed:
about pic

bethanyhaubert committed:
deleted letters

bethanyhaubert committed:
added post

bethanyhaubert committed:
added list item tags

bethanyhaubert committed:
updated listening to

bethanyhaubert committed:
typo

bethanyhaubert committed:
added posts

bethanyhaubert committed:
added diversity

bethanyhaubert committed:
edit

bethanyhaubert committed:
new theme

bethanyhaubert committed:
working theme

bethanyhaubert committed:
changed color

bethanyhaubert committed:
update xhr article

bethanyhaubert committed:
formatting xhr

bethanyhaubert committed:
working on format

bethanyhaubert committed:
again

bethanyhaubert committed:
new post

bethanyhaubert committed:
entry

bethanyhaubert committed:
rename

bethanyhaubert committed:
new post

bethanyhaubert committed:
1/3/18

bethanyhaubert committed:
again

bethanyhaubert committed:
today's shit post

bethanyhaubert committed:
new post

bethanyhaubert committed:
jan 10 post

bethanyhaubert committed:
dates wrong

bethanyhaubert committed:
date still wrong

bethanyhaubert committed:
journal

bethanyhaubert committed:
oops

bethanyhaubert committed:
january 31 journal entry

bethanyhaubert committed:
january 31 journal entry

bethanyhaubert committed:
new post

bethanyhaubert committed:
new post

bethanyhaubert committed:
new post

bethanyhaubert committed:
removed

bethanyhaubert committed:
new post

bethanyhaubert committed:
added CNAME file

bethanyhaubert committed:
Update CNAME

bethanyhaubert committed:
fix

bethanyhaubert committed:
Merge branch 'master' of github.com:bethanyhaubert/bethanyhaubert.github.io

bethanyhaubert committed:
new post

bethanyhaubert committed:
new entry

bethanyhaubert committed:
fix

bethanyhaubert committed:
new post

bethanyhaubert committed:
new theme

bethanyhaubert committed:
added new headers

bethanyhaubert committed:
Create CNAME

sumeetjain committed:
Replace some calls to site.baseurl with absolute_url filter

Example: A stylesheet is located at /assets/styles.css ... On the
homepage (which is located at the root path of '/'), the href for that
stylesheet could be href="assets/styles.css", because that relative path
is LUCKILY valid:

homepage's path + relative href = /assets/styles.css

But on the Aboute page, the relative path is UNLUCKILY not valid. The
About page is located at /about/ so:

about's path + relative href = /about/assets/styles.css

That's not where the stylesheet actually lives.

So we need the paths to that stylesheet to NOT BE RELATIVE.

In Liquid, {{ "a string" | filter_method }} means "process the given
string with the filter method"... e.g. {{ "beth haubert" | capitalize }}
would return "Beth Haubert".

So I replaced site.baseurl usages with {{ "some/path/styles.css" |
absolute_url }}, which returns
"https://bethhaubert.com/some/path/styles.css" ... So it's no longer a
relative path at all. Whether you're looking at the homepage or the
About page, you will always be asking the same absolute path for
stylesheets.

---

Note: The reason for this change is that, for some unknown reason,
site.baseurl was returning nil (empty string, actually) on production.
It should be returning "/" which would have worked fine. But instead of
debugging that, I worked around the issue instead.

sumeetjain committed:
Fix logo image URL

bethanyhaubert committed:
properly format talks and projects

bethanyhaubert committed:
fixed absolute_url links, update speaking & side projects

bethanyhaubert committed:
updated about page

bethanyhaubert committed:
first day at thoughtbot post

bethanyhaubert committed:
updates speaking engagements

bethanyhaubert committed:
fix

bethanyhaubert committed:
removes unneeded headers

bethanyhaubert committed:
Merge branch 'master' into update-speaking-engagements

bethanyhaubert committed:
Merge pull request #2 from bethanyhaubert/update-speaking-engagements

Update speaking engagements
